concepts/05 UI Components/DateBox/01 Overview/10 Key Features.md

```diff
@@ -1,5 +1,35 @@
+- **DateBox Types**    
+You can configure DateBox [type](/Documentation/ApiReference/UI_Components/dxDateBox/Configuration/#type) to integrate the component as a date, time, or date and time picker.
+
+- **Picker Configuration**    
+DateBox supports multiple [picker types](/Documentation/ApiReference/UI_Components/dxDateBox/Configuration/#pickerType), including calendar, list, and roller pickers. You can also implement native pickers that vary depending on user devices and platforms.
 
 - **Formatting**    
-DateBox [formats](/concepts/05%20UI%20Components/DateBox/04%20Value%20Formatting%20and%20Masked%20Input.md '/Documentation/Guide/UI_Components/DateBox/Value_Formatting_and_Masked_Input/') its value according to the application's locale. Use [predefined or custom](/concepts/Common/Value%20Formatting '/Documentation/Guide/Common/Value_Formatting/') formats.
\ No newline at end of file
+The DateBox component automatically formats values based on your application locale. You can also [specify display formats](/Documentation/Guide/UI_Components/DateBox/Value_Formatting_and_Masked_Input/) manually, including [predefined, Intl, and custom formats](/Documentation/Guide/Common/Value_Formatting/#Format_UI_Component_Values).
+
+- **Masked Input**    
+DateBox supports [masked input](/Documentation/ApiReference/UI_Components/dxDateBox/Configuration/#useMaskBehavior). When enabled, the component allows users to input only characters supported by the DateBox [display format](/Documentation/ApiReference/UI_Components/dxDateBox/Configuration/#displayFormat).
+
+- **Configurable Date and Time Ranges**    
+You can limit DateBox selection to a specific [date and time range](/Documentation/Guide/UI_Components/DateBox/Getting_Started_with_DateBox/#Set_the_Accepted_Date_Range). For instance, you can specify a new [Date](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date) object as the [minimum](/Documentation/ApiReference/UI_Components/dxDateBox/Configuration/#min) to disable date and time values in the past.
+
+- **Disabled Dates**    
+DateBox allows you to [disable individual dates](/Documentation/Guide/UI_Components/DateBox/Getting_Started_with_DateBox/#Disable_Specific_Dates). You can implement this feature to disable days such as holidays.
+
+- **Drop-Down Calendar Configuration**    
+You can customize [DateBox Calendar options](/Documentation/ApiReference/UI_Components/dxDateBox/Configuration/#calendarOptions) such as zoom level and week numbers.
+
+- **Form Support**    
```

Comment:
The only thing that I can say that we can merge Form support and Validation in one point of the list
Everything else is great!